tests/SubjectFilterTest.php

```
@@ -1,314 +1,397 @@
+use PHPUnit\Framework\TestCase;
 
 class SubjectFilterTest extends TestCase
 {
-    protected $filter;
+    protected SubjectFilter $filter;
 
-    protected function set_up()
+    protected function setUp(): void
     {
         $filter_factory = new FilterFactory();
         $this->filter = $filter_factory->newSubjectFilter();
```

PHPStan (2.1.54)

[error] Property Aura\Filter\SubjectFilterTest::$filter (Aura\Filter\SubjectFilter) does not accept Aura\Filter_Interface\SubjectFilterInterface.


(assign.propertyType)